src/object.c

```diff
@@ -862,7 +861,7 @@ size_t objectComputeSize(robj *o, size_t sample_size) {
         }
     } else if (o->type == OBJ_HASH) {
         if (o->encoding == OBJ_ENCODING_ZIPLIST) {
-            asize = sizeof(*o)+(ziplistBlobLen(o->ptr));
+            asize = sizeof(*o)+ zmalloc_size(o->ptr);
```

Comment:
```suggestion
            asize = sizeof(*o)+zmalloc_size(o->ptr);
```
Code style needs to be optimized, as does the above.

Reply:
Thank you for that advice. I just improved the code style.